.github/workflows/welcome-new-contributors.yaml

```diff
@@ -0,0 +1,28 @@
+name: Welcome new contributors
+
+on:
+  pull_request_target:
+    types:
+      - opened
+
+jobs:
+  welcome:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+    steps:
+      - uses: actions/first-interaction@v3
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          pr-message: |
+            🎉 **Welcome to the Kubeflow SDK!** 🎉
+
+            Thanks for opening your first PR! We're happy to have you as part of our community 🚀
+
+            **Here's what happens next:**
+
```

Comment:
Maybe it worths to also link the Kubeflow contributor guide, since it explains that commits must be singed: https://www.kubeflow.org/docs/about/contributing/